addons/editor-theme3/addon.json

```diff
@@ -231,6 +235,27 @@
         "input-color": "#202020",
         "text": "colorOnBlack"
       }
+    },
+    {
+      "name": "Material",
+      "id": "material",
+      "description": "A material color scheme with colors easy to tell apart",
+      "values": {
+        "motion-color": "#4C97FF",
+        "looks-color": "#9966FF",
+        "sounds-color": "#CF63CF",
+        "events-color": "#FFBF00",
+        "control-color": "#FFAB19",
+        "sensing-color": "#5CB1D6",
+        "operators-color": "#59C059",
+        "data-color": "#FF8C1A",
+        "data-lists-color": "#FF661A",
+        "custom-color": "#FF6680",
+        "Pen-color": "#0FBD8C",
+        "sa-color": "#29BEB8",
+        "input-color": "#202020",
+        "text": "colorOnBlack"
+      }
```

Comment:
Are you sure that these are intended values? It's just a copy of the Black preset.

Reply:
No...

Reply:
Fixed